client_src/js/system/server.options.js

```diff
@@ -0,0 +1,80 @@
+const serverDatabaseKeys = {
+  ALL: "Everything",
+  ALERTS: "Alerts",
+  CLIENT: "ClientSettings",
+  FILAMENT: "Filament",
+  HISTORY: "History",
+  PRINTERS: "Printers",
+  ROOMDATA: "RoomData",
+  SERVER: "ServerSettings",
+  USERS: "Users"
+};
+
+const serverActionsElements = {
+  OP_TIMELAPSE_SETUP: document.getElementById("setupTimelapseOctoPrint"),
+  LOG_DUMP_GENERATE: document.getElementById("logDumpGenerateBtn"),
+  RESET_DASHBOARD: document.getElementById("resetDashboardBtn"),
+  SAVE_SERVER_SETTINGS: document.getElementById("saveServerSettings"),
+  SAVE_CLIENT_SETTINGS: document.getElementById("saveClientSettingsBtn"),
+  RESTART_OCTOFARM: document.getElementById("restartOctoFarmBtn"),
+  UPDATE_OCTOFARM: document.getElementById("updateOctoFarmBtn"),
+  CHECK_OCTOFARM_UPDATES: document.getElementById("checkUpdatesForOctoFarmBtn")
+};
+
+const localStorageKeys = {
+  DASHBOARD_SETTINGS: "dashboardConfiguration"
+};
+
+// TODO clean up with a settings migration...
+// onlinePolling.seconds -> octoprintThrottle
+// timeout.apiRetry <-> timeout.apiRetryCutoff
+// filament.filamentCheck -> filamentManager.isFilamentSelected
```

Comment:
Could you add this to DevOps?

Reply:
Yeah no trouble

Reply:
I'll probably set them as tasks under the clean up one?